packages/retail-ui/components/Modal/Modal.tsx

```diff
@@ -196,6 +208,11 @@ export default class Modal extends React.Component<ModalProps, ModalState> {
     );
   }
 
+  private fixWidth = (rect: ClientRect | DOMRect) => this.setState({ width: rect.width });
+
+  private isThereScrollBar = (): boolean =>
+    !!this.containerNode && this.containerNode.clientWidth !== this.containerNode.offsetWidth;
```

Comment:
```suggestion
    !!this.containerNode && this.containerNode.scrollHeight > this.containerNode.clientHeight;
```

Такая проверка не будет работать c "оверлейными" скроллбарами: на мобилках, на маках, с кастомными какими-нибудь.

Reply:
Подтверждаю. Не работает :(
![Screen Recording 2019-05-23 at 11 46 17_17](https://user-images.githubusercontent.com/6397708/58231487-93151980-7d50-11e9-8b47-6080c359ad3e.gif)